Build failure for ScreenResultReporterTest

Newer versions of maven report the following error on build:

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:testCompile (default-testCompile) on project screensaver-core: Compilation failure: Compilation failure:
[ERROR] /home/CCRI/pertsemlidis/screensaver-master/core/src/test/java/edu/harvard/med/screensaver/service/screenresult/ScreenResultReporterTest.java:[764,61] incompatible types
[ERROR] required: int
[ERROR] found:    int[]
[ERROR] /home/CCRI/pertsemlidis/screensaver-master/core/src/test/java/edu/harvard/med/screensaver/service/screenresult/ScreenResultReporterTest.java:[843,61] incompatible types
[ERROR] required: int
[ERROR] found:    int[]

--- a/core/src/test/java/edu/harvard/med/screensaver/service/screenresult/ScreenResultReporterTest.java
+++ b/core/src/test/java/edu/harvard/med/screensaver/service/screenresult/ScreenResultReporterTest.java
@@ -770,7 +770,7 @@ public class ScreenResultReporterTest extends AbstractSpringPersistenceTest
                                                              "The positives",
                                                              "Overall",
                                                              "The overall",
-                                                             screenType);
+                                                             screenType)[0];
     assertTrue(count > 0);
 
     flushAndClear();
@@ -849,7 +849,7 @@ public class ScreenResultReporterTest extends AbstractSpringPersistenceTest
                                                              "The positives",
                                                              "Overall",
                                                              "The overall",
-                                                             screenType);
+                                                             screenType)[0];
     assertTrue(count > 0);
 
     ////////////

